cmake/SetupPackages.cmake

```diff
@@ -22,7 +22,7 @@ if (RAJA_ENABLE_CUDA)
       message(STATUS "Setting RAJA_ENABLE_EXTERNAL_CUB ON with CUDA_VERSION ${CUDA_VERSION_STRING} >= 11.")
     else()
```

Comment:
I was thinking the RAJA_ENABLE_EXTERNAL_CUB and the FindCUB.cmake module could also be removed. I'm not sure we want to use a version of CUB different than the CUDA toolkit we are using. So all this logic could be replaced with something like:

```
if (RAJA_ENABLE_CUDA)
  set(CUDAToolkit_ROOT ${CUDA_TOOLKIT_ROOT_DIR})
  find_package(CUDAToolkit 11 REQUIRED)

  blt_import_library(NAME cub
                                INCLUDES ${CUDAToolkit_INCLUDE_DIRS}
                                TREAT_INCLUDES_AS_SYSTEM ON
                                EXPORTABLE ON)

  if (RAJA_ENABLE_NVTX)
  ...
```

Reply:
Maybe someone else could weigh in as well, though.

Reply:
Its unlikely that we will need to use a cub/rocprim that's different than the one in the cuda/hip install. I suppose its possible that someone could wan tot try a newer version however so it could be nice to allow the user to set a path just in case.

Reply:
So maybe it would make more sense to default a variable like CUB_DIR to CUDA_TOOLKIT_ROOT_DIR and still maintain a FindCUB.cmake module that looks in CUB_DIR to find the header and imports a target with the include directory. I think that would be simpler than the RAJA_ENABLE_EXTERNAL_CUB logic.